### PR TITLE
Added service mesh patterns

### PR DESCRIPTION
**Pull request by Shivanshu97i**
**Description**
This is my first time contributing in open source, looking forward to more contributions.
Added the service mesh pattern in the project list. Took the image itself from the site provided. The description of the project is also taken from the site's readme section.

**Notes for Reviewers**
Signed-off-by: Shivanshu Singh singhshivanshu97@gmail.com

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by debo19 — CHANGES_REQUESTED** [review]
@Shivanshu97i I think the issue you are working on is [this](https://github.com/layer5io/.github/issues/5#issuecomment-924299120) . Please create the pull request in the [.github](https://github.com/layer5io/.github) repo not this one. 

Also, make sure to sign off your commits for the DCO test to pass. You can follow [these instructions](https://github.com/layer5io/.github/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).

**Comment by l5io** [comment]
🚀 Preview for commit 8a58866f7687841172f021668f2f0a182d46b56d at: https://614a30a8afede40859c21bbf--layer5ng.netlify.app

**Comment by warunicorn19** [comment]
@debo19 @Shivanshu97i let's keep this PR, we could include it in this readme as well.

**Comment by Shivanshu97i** [comment]
Signed-off-by: Shivanshu Singh singhshivanshu97@gmail.com

**Comment by debo19** [comment]
@Shivanshu97i commenting will not help :sweat_smile:. Follow these steps ( https://github.com/layer5io/layer5/pull/2209/checks?check_run_id=3667328476 ) to sign off in this PR. 
When you create the PR in .github (https://github.com/layer5io/.github) repo , follow this (https://github.com/layer5io/.github/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin) while commiting.

**Comment by Shivanshu97i** [comment]
@debo19 Sorry, that was dumb.😅

File: README.md [hunk 1]
```diff
@@ -37,6 +38,16 @@ Meshery facilitates adopting, configuring, operating and managing performance of
 </p>
 <br />
 <p style="clear:both;">
+<h2><a href="https://github.com/service-mesh-patterns">Service Mesh Patterns</a></h2>
+<a href="https://meshery.io"><img src="https://github.com/service-mesh-patterns/service-mesh-patterns/blob/master/.github/readme/images/service-mesh-pattern.png"
+style="margin:10px;" width="125px" 
+alt="Meshery - Service Mesh Patterns" align="left" /></a>
+
+Service mesh patterns help you get the most out of Linkerd, Consul, App Mesh, Istio, Kuma, Open Service Mesh or <a href="https://layer5.io/service-mesh-landscape">any service mesh</a>, including those using Envoy and use of WebAssembly. Each pattern can be used as a template and is customizable.
```
Comment (Nikhil-Ladha) [inline README.md:46]: Add the hyperlink for text `Service Mesh Patterns` here as well. Also, the name will be capitalized as shown.

**Comment by Shivanshu97i** [comment]
Should I close this PR since I created another PR in .github repo

**Comment by warunicorn19** [comment]
@Shivanshu97i we want to add service mesh patterns to the README in this repo as well, could you please just do the change that @Nikhil-Ladha has suggested above and remember to sign-off this time, then we can merge this PR.

**Comment by Shivanshu97i** [comment]
@warunicorn19 Ok sure!!

**Comment by warunicorn19** [comment]
Thank you

File: README.md (hunk 1 above)
Comment (Shivanshu97i) [reply README.md:46]: Ok will do it asap

**Comment by Shivanshu97i** [comment]
@warunicorn19 Should I close this PR??

**Comment by warunicorn19** [comment]
Yeah we can close this one.

**Pull request closed without merging**